runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContentsClient.java

```diff
@@ -66,7 +66,7 @@ public void didChangeThemeColor(int color) {
 
         @Override
         public void didStopLoading(String url) {
-            onPageFinished(url);
```

Comment:
Does the following call happens at the same thread? If true, this fix is ok.
XWalkWebContentsObserver::didStopLoading
InterceptNavigationDelegateImpl::shouldIgnoreNavigation